cardea/benchmark/task.py

```diff
@@ -0,0 +1,193 @@
+import json
+import os
+import pickle
+import shutil
+
+PIPELINES = ['Logistic Regression', 'K-Nearest Neightbors', 'Random Forest',
+             'Gaussian Naive Bayes', 'Multinomial Naive Bayes', 'XGB',
+             'Stochastic Gradient Descent', 'Gradient Boosting']
+
+PROBLEMS = ['LOS', 'Mortality', 'Readmission']
+
+# relative paths from the project root path to the target directories
+PIPELINE_DIR = './cardea/pipelines'
+VERIFIED_DIR = './benchmark/verified'
```

Comment:
I would use another approach (same as the reason earlier).

```python3
from import dirname

PIPELINE_DIR  = os.path.join(os.path.dirname(__file__), 'pipelines')
VERIFIED_DIR = os.path.join(os.path.dirname(os.path.dirname(__file__)), 'benchmark', 'verified')
```

Reply:
Current paths in task configurations are all relative paths from the root. Since we will share the benchmarking results, this design ensures that the tasks are runnable on others' devices. Are you suggesting we use absolute paths instead (which is more readable but not runnable on different devices)?

Reply:
I see your intention but the problem is this is difficult to trace. My suggestion would be to have the absolute path and the users should be following the same configuration of Cardea.

I imagine that `PIPELINE_DIR` will be defined in the same manner (in all cases) but the `VERIFIED_DIR` will change depending on the user. Is this the case you are referring to by 
> not runnable on different devices?

If that's what you mean, then I think that is not an issue, it can always be overriden by the user to specify their own path. But generally speaking, absolute paths are more maintainable and traceable as well.